docs/content/references/ts-sdk/kiosk/advanced-examples.mdx

```diff
@@ -14,7 +14,7 @@ const kioskClient = new KioskClient({
     client: new IotaClient({
         url: getFullnodeUrl('testnet'),
     }),
-    network: Network.TESTNET,
+    network: 'testnet',
```

Comment:
better to modify the import of `Network` to `@iota/iota-sdk/client,` instead of hardcoding the value. 
Same for the rest